helpers.sh

```diff
@@ -86,12 +86,12 @@ function set_base_lmp_version {
 	run git fetch origin --tags --quiet
 
 	# 3: Find our base LMP version based on the HEAD
-	export LMP_VER=$(git describe --tags --abbrev=0 HEAD)
+	export LMP_CACHE_VERSION=$(git describe --tags --abbrev=0 HEAD)
```

Comment:
```
-LMP_VER
+LMP_VERSION_SHORT
```
maybe this one is better

Reply:
yep. I hate to nitpick variable names, but I think `LMP_VERSION` (since it was already that) and `LMP_VERSION_SHORT` or `LMP_VERSION_NUM` would be better.

Reply:
changed to `LMP_VERSION_NUM`